### ISSUE TITLE
`piuparts` not available on 24.04

### ISSUE THREAD

**Issue by jpm-canonical** [canonical/ubuntu-maintainers-handbook#130]
https://github.com/canonical/ubuntu-maintainers-handbook/blob/d654e77a4180571b85aae7689d0424ebbc4b0e8f/Setup.md?plain=1#L34

Getting an error when trying to set up an Ubuntu Desktop 24.04 development machine:
```
E: Package 'piuparts' has no installation candidate
```

**Comment by basak** [comment]
It does appear to be missing. I suggest just skipping using it for now. Its use is fairly rare.

I filed https://bugs.launchpad.net/ubuntu/+source/piuparts/+bug/2073768 to track this.

For the handbook, I don't know if it's appropriate to remove this or not.

**Comment by basak** [comment]
> I don't know if it's appropriate to remove this or not.

Indeed, nothing in the handbook actually uses it, so perhaps it's easiest just to drop it from the handbook if/until instructions on how to use this tool are written, and it is reinstated in Noble.

**Comment by mitchdz** [comment]
I vote to just drop it from the handbook.

**Comment by lvoytek** [comment]
Just to note, `piuparts` has returned to the archive in oracular + plucky, but is still missing from noble. Since nothing in the handbook uses it I agree with dropping it. Adding a pr to fix that
